cypress/integration/visitorCanSeeMultipleLanguages.spec.js

```diff
@@ -0,0 +1,17 @@
+describe("Visitor can see home button in", () => {
+  before(function() {
+    cy.visit("http://localhost:3001");
```

Comment:
You need to stub out the root path or you'll end up with a warning. We don't want that. I had to fix that in most tests earlier today and asked for everyone on the team to review that.

Reply:
fixed